src/functions/services/clientCredentialsService/clientCredentialsService.test.ts

```diff
@@ -79,17 +120,64 @@ describe("Client Credentials Service", () => {
     });
   });
 
+  describe("Validate credential request credentials", () => {
+    describe("redirect_uri validation", () => {
+      describe("Given stored redirect_uri is not present", () => {
+        it("Returns a log", () => {
+          mockStoredClientCredentials.redirect_uri = "";
```

Comment:
This can be removed or maybe set to null or undefined now that the interface has been updated to have this field as an optional parameter